models/event_filters/stripe_charges.sql

```diff
@@ -5,7 +5,7 @@ with charges as (
 )
 
 select
-  id,
+  data__object__id as id,
```

Comment:
Makes sense. One other thing I noticed is that 'charge.refunded' events were coming through with 'funded' in the `result`... I'm not familiar with `trim()` in Redshift, but there might be some strange behavior there